src/index.js

```diff
@@ -194,6 +194,11 @@ class Dropzone extends React.Component {
       const acceptedFiles = []
       const rejectedFiles = []
 
+      if (isIeOrEdge() && evt.dataTransfer && !isFileList(Array.from(evt.dataTransfer.items))) {
+        rejectedFiles.push(...fileList)
```

Comment:
I'm not so sure that we should reject in this case, because what you actually dropped are not files (technically Edge says they are, when in fact they are urls). I think we should only reject when the accepted mime type does not match with the files that are dropped.

Reply:
You're also missing the first `!isFileList(fileList)`:
```js
if (
  !isFileList(fileList) ||
  (isIeOrEdge() && evt.dataTransfer && !isFileList(Array.from(evt.dataTransfer.items)))
) {
  return
}
```